packages/embedder/ai_models/AbstractEmbeddingsModel.ts

```diff
@@ -42,70 +69,117 @@ export abstract class AbstractEmbeddingsModel extends AbstractModel {
   }
 
   async chunkText(content: string) {
-    const AVG_CHARS_PER_TOKEN = 20
+    const AVG_CHARS_PER_TOKEN = 4
```

Comment:
the reason this was boosted so high was because we want to try REAL HARD to avoid chunking.  Why chunk if you don't have to?